homework.md

```diff
@@ -4,110 +4,110 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+Black
```

Comment:
Hi Sima I think that would be white :D

Reply:
I have no idea how I messed that answer up but thanks.